### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by scarlehoff**
As explained in #6 

Since now we have two possible problems in the readme I've added a "common problems" header to it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,27 @@ This script does three things:
...
+Common issues
+-------------
+
 In case of a conflict between local and conda-installed packages, use the environment flag `PYTHONUSERBASE`:
 
 ```bash
 export PYTHONUSERBASE=intentionally-disabled
 ```
+
+If the installation of the `nnpdf` package hangs or takes a long time, a solution is to use the `libmamba` solver as [explained here].
```
Comment (scarlehoff) [inline README.md:31]:
```suggestion
If the installation of the `nnpdf` package hangs or takes a long time, a solution is to use the `libmamba` solver as [explained here](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community).
```

File: README.md [hunk 2]
```diff
@@ -6,19 +6,27 @@ This script does three things:
...
+-------------
+
 In case of a conflict between local and conda-installed packages, use the environment flag `PYTHONUSERBASE`:
 
 ```bash
 export PYTHONUSERBASE=intentionally-disabled
 ```
+
+If the installation of the `nnpdf` package hangs or takes a long time, a solution is to use the `libmamba` solver as [explained here](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community).
+Note that from September '23, `libmamba` will be the default solver in conda so after this date updating conda `conda update -n base conda` should be enough.
```
[the remark this answers is not in the document]
Comment (RoyStegeman) [reply README.md:32]: Just so people can compare to their own conda version to see if this can explain their problem

**Comment by RoyStegeman** [comment]
Perhaps we should indeed remove the private channel as well? It's no longer needed and even makes it inconsistent with the description of this script in the readme.

**Comment by scarlehoff** [comment]
For now it will just be a secret channel that people will have :P

**Pull request merged**